src/decision/src/FinalDecision.cpp

```diff
@@ -24,8 +24,8 @@ FinalDecision::FinalDecision(int argc, char **argv, std::string node_name) {
     gps_subscriber = public_nh.subscribe(gps_decision_topic_name, queue_size, &FinalDecision::gpsCallBack, this);
 
     // Setup Publisher(s)
-    std::string twist_topic = public_nh.resolveName("twist");
-    twist_publisher = nh.advertise<geometry_msgs::Twist>(twist_topic, queue_size);
+    std::string twist_topic = public_nh.resolveName("cmd_vel");
```

Comment:
Is `resolveName` needed with a public nodehandle? I think it just brings it to the global namespace